package.json

```diff
@@ -57,7 +57,7 @@
     "@types/google.maps": "^3.45.3",
     "@types/hogan.js": "^3.0.0",
     "@types/qs": "^6.5.3",
-    "algoliasearch-helper": "^3.9.0",
+    "algoliasearch-helper": "^3.10.0",
```

Comment:
Was this upgrade required for this PR? It changes a snapshot that seems unrelated to the Search Client v5 support.

Reply:
Yes, both changes were done in the same version